CHANGELOG.md

```diff
@@ -2,6 +2,19 @@
 
 ## Unreleased
 
+## v0.74.0
+
+This Splunk OpenTelemetry Collector release includes changes from the [opentelemetry-collector v0.74.0](https://github.com/open-telemetry/opentelemetry-collector/releases/tag/v0.74.0) and the [opentelemetry-collector-contrib v0.74.0](https://github.com/open-telemetry/opentelemetry-collector-contrib/releases/tag/v0.74.0) releases where appropriate.
+
+### 💡 Enhancements 💡
+- [Relocate agent codebase into pkg/signalfx-agent](https://github.com/signalfx/splunk-otel-collector/pull/2717)
+- [Tanzu Tile implementation and documentation](https://github.com/signalfx/splunk-otel-collector/pull/2726)
```

Comment:
Once released, would this trigger an action to create the Tanzu tile.yml file as part of our release artifacts or hosted elsewhere?

Reply:
AFAICT, the tile is not yet included in the release pipeline or published anywhere. Are there specific requirements for the release?  cc @crobert-1

Reply:
In the past, we didn't publish specific artifacts for cloud foundry, just the release instructions. The tile gets pushed to the Tanzu marketplace directly, and some of the tile code grabs the latest collector release binaries. So we should be good - @crobert-1 please confirm?

Reply:
_I see this change is already merged, but I'll still answer._ 

This does not currently push the tile anywhere. It will only currently be saved as an artifact of a github action for the purposes of debugging.